fern/pages/docs/building-your-docs/customizing-slugs.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Customizing slugs within your site
+---
+
+By default, Fern generates the slug of a page based on the navigation structure in the `docs.yml` file. 
+
+<AccordionGroup>
+<Accordion title="Example without tabs">
+```yaml docs.yml
+instances:
+  - url: plantstore.docs.buildwithfern.com
+  
+navigation:
+  - section: Get Started
```

Comment:
want to do syntax highlighting to call out the line numbers that form the slug? 14 and 16 in this example.

34, 36, and 38 in the next example.